opentelekomcloud/services/elb/v3/resource_opentelekomcloud_lb_policy_v3.go

```diff
@@ -0,0 +1,265 @@
+package v3
+
+import (
+	"context"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	golangsdk "github.com/opentelekomcloud/gophertelekomcloud"
+	"github.com/opentelekomcloud/gophertelekomcloud/openstack/elb/v3/policies"
+	"github.com/opentelekomcloud/gophertelekomcloud/openstack/elb/v3/rules"
+	"github.com/opentelekomcloud/terraform-provider-opentelekomcloud/opentelekomcloud/common"
+	"github.com/opentelekomcloud/terraform-provider-opentelekomcloud/opentelekomcloud/common/cfg"
+	"github.com/opentelekomcloud/terraform-provider-opentelekomcloud/opentelekomcloud/common/fmterr"
+)
+
+func ResourceLBPolicyV3() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceLBPolicyV3Create,
+		ReadContext:   resourceLBPolicyV3Read,
+		UpdateContext: resourceLBPolicyV3Update,
+		DeleteContext: resourceLBPolicyV3Delete,
+
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Computed:     true,
+				ValidateFunc: validation.StringLenBetween(0, 0xff),
+			},
+			"description": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Computed:     true,
+				ValidateFunc: validation.StringLenBetween(0, 0xff),
+			},
+			"project_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"action": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					"REDIRECT_TO_POOL", "REDIRECT_TO_LISTENER",
+				}, false),
+			},
+			"listener_id": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.IsUUID,
+			},
+			"redirect_listener_id": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validation.IsUUID,
+			},
+			"redirect_pool_id": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validation.IsUUID,
+			},
+			"position": {
+				Type:         schema.TypeInt,
+				Optional:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.IntBetween(1, 100),
+			},
+			"rules": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 10,
```

Comment:
Rules should be `Computed` - they can be created with policy rules API too.
Also, I would use `TypeSet` instead of `TypeList`, so reordering won't break refresh.